test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -350,33 +373,17 @@ public void CanUpgradeRequestWithConnectionKeepAliveUpgradeHeader()
                 using (var socket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
                 {
                     socket.Connect(new IPEndPoint(IPAddress.Loopback, host.GetPort()));
-                    socket.Send(Encoding.ASCII.GetBytes("GET"));
```

Comment:
Do we still have a test that verifies that a bad/incomplete request doesn't cause any logs with a higher level than information?